java/testing/TestJavaBindings.java

```diff
@@ -0,0 +1,30 @@
+import io.github.f3d_app.f3d.*;
+
+public class TestJavaBindings {
+
+  static {
+    if (System.getProperty("os.name").startsWith("Windows"))
+    {
+      // On Windows, preload the OpenGL library
+      // This ensures the OpenGL used is the one in the working directory if any.
+      // In practice, it is used in F3D CI to run the test using Mesa, it's not required for production.
+      System.loadLibrary("opengl32");
```

Comment:
no LD_PRELOAD equivalent on windows ?

Reply:
nop, the variable `PATH` is used. But when `f3d-java.dll` is loaded by JVM, it uses this order:

- The directory from which the application loaded.
- The system directory. Use the [GetSystemDirectory](https://learn.microsoft.com/en-us/windows/desktop/api/sysinfoapi/nf-sysinfoapi-getsystemdirectorya) function to get the path of this directory.
- The 16-bit system directory. There is no function that obtains the path of this directory, but it is searched.
- The Windows directory. Use the [GetWindowsDirectory](https://learn.microsoft.com/en-us/windows/desktop/api/sysinfoapi/nf-sysinfoapi-getwindowsdirectorya) function to get the path of this directory.
- The current directory.
- The directories that are listed in the PATH environment variable. Note that this does not include the per-application path specified by the App Paths registry key. The App Paths key is not used when computing the DLL search path.

Since `open32.dll` is found in the system directory, this one is used, except if the library is already loaded in the application (that's why I load it manually)

Reply:
ok